src/main/java/org/traccar/protocol/T800xProtocolDecoder.java

```diff
@@ -163,7 +163,12 @@ protected Object decode(
 
         boolean positionType = type == MSG_GPS || type == MSG_GPS_2 || type == MSG_ALARM || type == MSG_ALARM_2;
         if (!positionType) {
-            sendResponse(channel, header, type, index, imei, 0);
+
+            if (header == 0x2323) {
+                sendResponse(channel, header, type, 0x0001, imei, 0);
```

Comment:
Can we send 1 for other devices as well?

Reply:
according the official documentation of the manufacturer, we believe that all devices use header 0x2323 will need to receive the serial `1`.

Reply:
I meant devices with other headers?

Reply:
I guess no, starting from header 0x2525 the serial will be computed depending on device info

We are testing some devices and I can confirm that it will not be the case in the following headers:

0x2525 gen 1 and gen 2
0x2626
0x2727

Reply:
Hello Anton, it's not possible; it only applies to devices with header ```0x2323```. In other cases, you need to respond with the serial number that comes in the device's payload. The documentation regarding other headers from Topflytech can be found in the description of this [pull request](https://github.com/traccar/traccar/pull/5191).

Reply:
OK, can we do something like this then: `header == 0x2323 ? 1 : index`

Reply:
Yeah sure, I'll add it

Reply:
done 👍